backend/module/chaos/experimentation/xds/rtds_config.go

```diff
@@ -0,0 +1,5 @@
+package xds
+
+type RTDSConfig struct {
```

Comment:
I'm not sure if there's much value in having this be in its own file

Reply:
will move it